src/main.ts

```diff
@@ -152,6 +152,10 @@ export default class SlatedPlugin extends Plugin {
 
   private async loadSettings(): Promise<void> {
     this.settings = settingsWithDefaults(await this.loadData());
+
+    if (shouldConfigureGlobalMoment(this.app)) {
```

Comment:
This makes sure that the locale is configured when the plugin starts up (oops, totally forgot this in the last PR)